fio/fio_run

```diff
@@ -254,20 +256,154 @@ report_info()
 	echo $1 >> /tmp/log
 }
 
+convert_unit()
+{
+	unit=$1
+	base=1
+
+	if [[ $unit == "KiB/s" ]]; then
+		base=1024
+	elif [[ $unit == "MiB/s" ]]; then
+		base=`echo 1024*1024 |bc`
+	elif [[ $unit == "GiB/s" ]]; then
+		base=`echo 1024*1024*1024 |bc`
```

Comment:
Maybe to make this more readable, you can express KiB/s as 2^10, MiB/s as 2^20, etc.